tests/openwisp2/settings.py

```diff
@@ -164,9 +164,13 @@
 OPENWISP_ORGANIZATION_OWNER_ADMIN = True
 OPENWISP_USERS_AUTH_API = True
 
-CELERY_TASK_ALWAYS_EAGER = True
-CELERY_TASK_EAGER_PROPAGATES = True
-CELERY_BROKER_URL = 'memory://'
+if not TESTING:
+    CELERY_BROKER_URL = os.getenv('REDIS_URL', 'redis://localhost/1')
+else:
+    CELERY_TASK_ALWAYS_EAGER = True
+    CELERY_TASK_EAGER_PROPAGATES = True
+    CELERY_BROKER_URL = 'memory://'
```

Comment:
Here's the problem with this change: it requires to upgrade the README instructions for users who want to run this app locally.
We could do that, but why even bother? I think we can just keep the eager mode, this also makes it easier for users to try this app. Maybe let's just add a comment above these settings like:

```
# the following 3 settings are for development only (don't use in production)
```